contracts/oracle/Makefile

```diff
@@ -1 +1,8 @@
 include ../Makefile-common.mk
+
+deploy: 
+	near deploy i-am-human-staging.testnet ../res/oracle_sbt.wasm
+deploy-init:
+	near deploy i-am-human-staging.testnet ../res/oracle_sbt.wasm
+	near call i-am-human-staging.testnet new '{"authority": "zqMwV9fTRoBOLXwt1mHxBAF3d0Rh9E9xwSAXR3/KL5E=", "metadata": {"spec": "sbt-1.0.0", "name": "Oracle v1", "symbol": "Oracle-SBT"}, "registry": "registry-1.i-am-human.testnet", "claim_ttl": 3600, "admin": "i-am-human-staging.testnet"}' --accountId i-am-human-staging.testnet
```

Comment:
This should not be in a Makefile. Please add info to README (under testnet section), with info about address, ttl and authority pub key.

Reply:
I added it here since it's a bit hard to deploy it every time without the script. Shouldn't we have an easier way to do it? Like a shell script or sth

Reply:
yes, script, but not a makefile, because we may deploy to many different contracts.